tests/Repositories/SettingsConfigRepositoryTest.php

```diff
@@ -20,4 +20,23 @@ public function testDotNotation()
 
         $this->assertEquals('the value', config('test.dot.notation'));
     }
+
+    public function testCachesValueSimple()
+    {
+        Setting::create([
+            'key' => 'cache-test',
```

Comment:
To keep reference with the Collections code could you change `'cache-test'` by smoething like `'collections.6.obfuscate'` and add a comment to reference the class were it is used.